src/test/java/peoplesoft/model/person/PersonTest.java

```diff
@@ -91,4 +91,51 @@ public void equals() {
         editedAlice = new PersonBuilder(ALICE).withTags(VALID_TAG_HUSBAND).build();
         assertFalse(ALICE.equals(editedAlice));
     }
+
+    @Test
+    public void isDuplicate() {
+        // same values -> returns true
+        Person aliceCopy = new PersonBuilder(ALICE).build();
+        assertTrue(ALICE.isDuplicate(aliceCopy));
+
+        // same object -> returns true
+        assertTrue(ALICE.isDuplicate(ALICE));
+
+        // null -> returns false
+        assertFalse(ALICE.isDuplicate(null));
+
+        // different type -> returns false
+        assertFalse(ALICE.isDuplicate(5));
+
```

Comment:
these can be removed as well if you do decide to have `isDuplicate()` take a `Person` instead of an `Object`

```suggestion
```